templates/web/base/waste/bulky/summary.html

```diff
@@ -110,7 +110,11 @@ <h3 class="summary-section--heading">Items to be collected</h3>
       END %]
       <dl>
         <dt>Price</dt>
-        [% payment = cobrand.bulky_total_cost(data) %]
+        [% IF problem.get_extra_field_value('payment') %]
```

Comment:
I think I'd rather always call this if `problem` is set at all (as then payment always should be present), so as not to fall back onto something that might theoretically be an incorrect value. Unless there's quite possibly a reason to do this I've missed.